src/compute-pr-actions.ts

```diff
@@ -136,28 +136,40 @@ function canBeMergedNow(info: PrInfo): boolean {
         return false;
     }
 
-    return hasFinalApproval(info);
+    return hasFinalApproval(info).approved;
 }
 
-function hasFinalApproval(info: PrInfo): boolean {
+function hasFinalApproval(info: PrInfo) {
+    let approved = false
+    let requiredApprovalBy = "DT Maintainers"
+
     if (info.dangerLevel === "ScopedAndTested") {
         if (info.popularityLevel === "Well-liked by everyone") {
-            return !!(info.approvalFlags & (ApprovalFlags.Maintainer | ApprovalFlags.Owner | ApprovalFlags.Other));
+            approved = !!(info.approvalFlags & (ApprovalFlags.Maintainer | ApprovalFlags.Owner | ApprovalFlags.Other));
+            requiredApprovalBy = "type definition owners, DT maintainers or others"

```

Comment:
"or others"  is up for word smithing IMO

It reads like this allows rando third parties who haven't added themselves to the top of the DT file, which is reasonable. I'm just not sure how to prime this as "anyone who cares"?